### PR TITLE
added default example

### PR DESCRIPTION
**Pull request by ajsingh273**

### CODE REVIEW

File: _includes/swagger.html [hunk 1]
```diff
@@ -1,41 +1,76 @@
...
+  <script src='http://petstore.swagger.io/lib/jquery.slideto.min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/jquery.wiggle.min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/handlebars-2.0.0.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/underscore-min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/backbone-min.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/swagger-ui.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/highlight.7.3.pack.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/marked.js' type='text/javascript'></script>
+  <script src='http://petstore.swagger.io/lib/swagger-oauth.js' type='text/javascript'></script>
```
Comment (calumhalcrow) [inline _includes/swagger.html]: You still need to add these files to the project and serve them as part of the app. Don't load them from a 3rd party unless that URL intended to act as a CDN (and even then, do so with caution). Ideally, you shouldn't need an internet connection to run the app locally.

File: _includes/swag.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<body class="swagger-section">
+<div id="message-bar" class="swagger-ui-wrap">&nbsp;</div>
+<div id="swagger-ui-container" class="swagger-ui-wrap"></div>
+</body>
```
Comment (calumhalcrow) [inline _includes/swag.html]: Might as well call it swagger.html.

**Comment by calumhalcrow** [comment]
Yeah, I think this approach will work. It's not fully working yet though. When you try to use it, you get an `unknown_parameters": ["keyword"]` error. If you can make it work fully with the trade events endpoint, I'll merge this proof of concept so that Erik/Stuart can see.

**Comment by ajsingh273** [comment]
@calumhalcrow I made the changes you suggested. Also added a v2 Trade Events json file, which passed the error checker. 

But I am still getting a Response error: 
{
  "error": "Unauthorized" 
}

I think the apiKey is assigned correctly [here](https://github.com/GovWizely/developerportal/blob/aj/new-swagger-ui/_includes/swagger-ui.html)

Is there anything you see that might be causing this? Thanks.

**Comment by calumhalcrow** [comment]
@ajsingh273 use the browser's dev tools to examine the request:

screen shot 2015-06-02 at 11 51 26 am

The screenshot shows two things:
- The request doesn't pass the api_key in the query string.
- The request goes to production, even though I'm running the code in development.

You're going to get a 401 error if you don't specify the api_key. I'm going to have a look at this code now to see if I can fix it.

**Comment by jamesisaacs** [comment]
screen shot 2015-06-03 at 9 46 00 am

Looks like you guys fixed this. I'll merge this in for staging.

**Pull request merged**